lib/stripe/api_operations/request.rb

```diff
@@ -66,6 +67,7 @@ def execute_resource_request_stream(method, url,
           [resp, opts_to_persist]
         end
 
+        # TODO: (major)
```

Comment:
What is this TODO for?

Reply:
We often grep for `TODO: (major)` when cutting a new major release but we somehow missed this comment the last time we did a stripe-ruby major? So I added the magic string in hopes of it not getting lost.

Reply:
Oh I didn't read the entire comment below. Makes sense!